Whitepapers.md

```diff
@@ -564,14 +575,15 @@
 | Gimmer | Gimmer - Bitcoin bot | https://token.gimmer.net/assets/docs/gimmer-wp-en.pdf | https://gimmer.net/ | N/P* |
 | Ginger Coin | Ginger Coin | https://ginger-coin.com/files/Whitepaper.pdf | https://ginger-coin.com/ | N/P* |
 | GISC LoanCoin | GISC LoanCoin - The official token for GIS Corp | https://crypto.giscorponline.com/wp-content/uploads/2017/08/GlobalInvestmentServicesCorp-GISCLoanCoinWhitepaper.pdf | https://crypto.giscorponline.com/ | N/P* |
+| Givacoin | Consumer Fundraising on Blockchain | https://www.givacoin.io/wp-content/uploads/2017/10/GivaCoin-Whitepaper-v1.0-ENG.pdf | https://www.givacoin.io/ | N/P* |
 | Gizer | Gizer - The Global Gaming Network | https://tokensale.gizer.io/wp-content/uploads/2017/09/One-Pager.pdf | https://gizer.io/ | N/P* |
 | Gladius | Gladius - Blockchain-driven cyber protection network | https://gladius.io/pdf/gladius-whitepaper.pdf | https://gladius.io/ | N/P* |
 | Global Academy Place | Global Academy Place | https://globalacademyplace.com/wp-content/uploads/2017/09/White-paper-GAP.pdf | https://globalacademyplace.com/ico/ | N/P* |
 | Global Crypto Bank | Global Crypto Bank | http://globalbank.io/docs/WP_v5.3.pdf | http://globalbank.io/ | N/P* |
 | Global Jobcoin | Global Jobcoin - Global Crypto Currency for Jobs - Globaljobcoin.com | https://www.globaljobcoin.com/media/files/documents/GJC_Whitepaper.pdf | https://www.globaljobcoin.com/ | N/P* |
 | Global Metrology Project | GLOBAL METROLOGY PROJECT - Initial Coin Offering | https://gmp.im/images/docs/GMP_Project_WhitePapper_EN.pdf | https://gmp.im/ | N/P* |
 | Globetrotter Ecosystem | GLOBETROTTER CRYPTOCURRENCY | https://docs.wixstatic.com/ugd/e8cc17_efa02d0e8c714bbc8911e29a312deb00.pdf | https://www.globetrottercoin.com/ | N/P* |
-| Gluon | GLU Token | https://www.glutoken.com/wp-content/uploads/2017/08/Gluon-Whitepaper.pdf | https://www.glutoken.com/ | N/P* |
```

Comment:
We should move that to Skip.md

Reply:
This move in Skip when I delete in Whitepaper